src/cinder/gl/draw.cpp

```diff
@@ -880,6 +1052,7 @@ void drawSolidCircle( const vec2 &center, float radius, int numSegments )
 		if( normals )
 			normals[s+1] = vec3( 0, 0, 1 );
 		t += tDelta;
+		cout << texCoords[s+1] << endl;
```

Comment:
whoops! I'll take care of that. Also, I'll update the Stroked Rounded Rect as well.